app/views/shared/check_answers/_merits.html.erb

```diff
@@ -1,400 +1,32 @@
-              question: t(".items.linked_children"),
-              answer: linked_children_names(proceeding),
-              read_only:,
-            ) %>
-      </dl>
-    <% end %>
-    <% if proceeding.specific_issue %>
-      <dl class="govuk-summary-list govuk-!-margin-bottom-2">
-        <%= check_answer_link(
-              name: "#{proceeding.id}_specific_issue".to_sym,
-              url: providers_merits_task_list_specific_issue_path(proceeding),
-              question: t(".items.specific_issue"),
-              answer: proceeding.specific_issue.details,
-              read_only:,
-            ) %>
-      </dl>
-    <% end %>
-    <% if proceeding.prohibited_steps %>
-      <dl class="govuk-summary-list govuk-!-margin-bottom-0 print-remove-border">
-        <%= check_answer_link(
-              name: "#{proceeding.id}_prohibited_steps".to_sym,
-              url: providers_merits_task_list_prohibited_steps_path(proceeding),
-              question: t(".items.prohibited_steps"),
-              answer: yes_no(proceeding.prohibited_steps.uk_removal),
-              read_only:,
-              no_border: true,
-            ) %>
-      </dl>
-      <% unless proceeding.prohibited_steps.uk_removal %>
-        <div class="govuk-body"><%= proceeding.prohibited_steps.details %></div>
-      <% end %>
-      <%= render "shared/check_answers/section_break" %>
-    <% end %>
-    <% if proceeding.vary_order %>
-      <dl class="govuk-summary-list govuk-!-margin-bottom-2">
-        <%= check_answer_link(
-              name: "#{proceeding.id}_vary_order".to_sym,
-              url: providers_merits_task_list_vary_order_path(proceeding),
-              question: t(".items.vary_order"),
-              answer: proceeding.vary_order.details,
-              read_only:,
-            ) %>
-      </dl>
-    <% end %>
-    <% if proceeding.opponents_application %>
-      <dl class="govuk-summary-list govuk-!-margin-bottom-2 govuk-!-margin-top-2">
-        <%= check_answer_link(
-              name: "#{proceeding.id}_opponents_application".to_sym,
-              url: providers_merits_task_list_opponents_application_path(proceeding),
-              question: t(".items.opponents_application_question"),
-              answer: yes_no(proceeding.opponents_application.has_opponents_application),
-              read_only:,
-            ) %>
-        <% if proceeding.opponents_application.reason_for_applying %>
-          <%= check_answer_link(
-                name: "#{proceeding.id}_opponents_application".to_sym,
-                url: providers_merits_task_list_opponents_application_path(proceeding),
-                question: t(".items.opponents_application_reason"),
-                answer: proceeding.opponents_application.reason_for_applying,
-                read_only:,
-              ) %>
-        <% end %>
-      </dl>
-    <% end %>
-  </section>
-<% end %>
+<%# TODO: The gateway evidence may not exist for an application transitioning to MP feature %>
```

Comment:
I think the TODO can be removed, multi proceedings have been live for some time now